README.md

```diff
@@ -96,6 +96,10 @@ Click below to see the demo in a web browser:<br>
 2. `npm install`
 3. `ng build atft`<br>
 
+### Run Tests
+
+- `npm test`
```

Comment:
I assume that command is:
```ng test```
or 
```npm run test```

Reply:
In fact [`npm test`](https://docs.npmjs.com/cli/test.html) is equivalent to `npm run test`, and the latter is set to `ng test` in our package.json. So I guess `npm test` as the most abstract solution there is fine.

Reply:
In case of ```npm test```
I have an error - 'Error: no test specified' :(


```
>npm test
npm info it worked if it ends with ok
npm info using npm@6.9.0
npm info using node@v12.4.0
npm timing npm Completed in 75ms
npm info ok
npm info it worked if it ends with ok
npm info using npm@6.11.3
npm info using node@v12.4.0
npm info lifecycle atft@1.1.0~pretest: atft@1.1.0
npm info lifecycle atft@1.1.0~test: atft@1.1.0

> atft@1.1.0 test C:\Users\dm\github\angular-template-for-threejs\dist\atft
> echo 'Error: no test specified'

'Error: no test specified'
npm info lifecycle atft@1.1.0~posttest: atft@1.1.0
npm timing npm Completed in 174ms
npm info ok

````

Reply:
@makimenko Oh you ran it from the `dist/atft` directory? Run it from `./`. I think the assumption in https://github.com/makimenko/angular-template-for-threejs#development is running it from `./` for all entries. Might want to clarify that.

Reply:
> Oh you ran it from the `dist/atft` directory?

Yep, my bad. All good from the root folder. Thanks